### DISCUSSION TITLE
[上古问题+重磅消息]1996（？）年大宇资讯（信息）的FTP中放出的CRPG.EXE的存档修复工具是干啥用的 + 仙剑DOS试玩版 + 《风起云涌》官方MIDI出土

### DISCUSSION THREAD

**Discussion by miaocanada** [palxex/palresearch#12] [General]
最近在知乎上码字一篇游戏meta细节考（主要是跟游戏版次、修正档以及音乐配置相关的，将这些相关资料整理出来以飨后来人），发现早期的大宇资讯网站中曾放出一个仙剑奇侠传DOS版存档修复工具CRPG.exe，但我找不到其他相关细节了。只想问一下这个存档修复工具是干啥用的，会在什么情况下造成存档损坏才能修复。不甚感谢。
细节1：
http://web.archive.org/web/19980209082350/http://www.softstar.com.tw/serve/ftp/ssdemo.html
细节2（文件下载，找到“仙剑奇侠传DOS”的“修复档”便是）：
http://web.archive.org/web/20031204202201/http://www.joypark.com.tw/DOWNLOAD/default.asp 

明显这和修正档pal02.exe中的cvsave.exe不太一样，后者是将存档修正。

另外有一个请求，如果各位大大还有磁盘版仙剑的 .img或者其他格式镜像，和/或初版CD-ROM镜像（含有龙门防毒NVC-0004.EXE）的，望告知，十分感谢。

**Comment by miaocanada** [comment]
https://zhidao.baidu.com/question/76952754.html?si=6
抱歉我火星了，找到答案了，实在抱歉。

**Comment by miaocanada** [comment]
另外问一下磁盘版有无龙门防毒（NVC-0004.EXE），以及磁盘版的主程序命名为pal.exe还是pals.exe。目前只知道修正档中有pals.exe是针对磁盘版的。

**Comment by palxex** [comment]
挺久之前[在gamerhome上回复过其中些许](http://gamerhome.com/bbs/forum.php?mod=viewthread&tid=234469)。时光久远，防丢起见，掐头去尾节略于此。别者我亦不知。
```
一、《仙剑奇侠传》DOS版
    1.台湾发行版
        1)磁盘版(1995年6月)
            大磁盘版，12张5寸盘，游戏手册和主题海报。
            小磁盘版，10张3.5寸盘。内容无区别。
            未解压18M，解压后31M。
            问:原磁盘版的认证方式是什么？  答:色块(palxex)
            问:磁盘版有没有存档经验值Bug？  答:有。且磁盘版没有出过修正版。(palxex)

        2)光盘版
            初版，1995年6月，有存档经验值Bug。问:认证方式是数光盘上的轨道数是不是14？  答: 数，9，8音1数据。(palxex)
            修正版，1995年7月。问:除数轨道之外还有颜色认证？  答:否，但是附带的磁盘升级版主程序(PALS.exe)则只检测色块。(palxex)
            (以上来自本论坛仙痴描述)
            光盘版有音轨。(palxex解答)
            音轨曲目如下，8首。
                1 - 情怨
                2 - 蝶舞春园
                3 - 雨
                4 - 白河寒秋
                5 - 桃花幻梦
                6 - 云谷鹤风
                7 - 蝶恋
                8 - 比武招亲
            包装内盒黑色，盘盒卡纸为折叠多连的彩色歌纸。
            问:光盘版和磁盘版除认证方式外还有何区别？
            问:光盘版找不到音轨时会自动替换MIDI么？  答:磁盘版貌似是不会去播放CD的，未确认。未找到会替换为对应的RIX/MIDI，根据install的设定选择。(palxex)

        3)官方补丁
            jorpark.com.tw上有两个，一个“更新档”，一个“修复档”。
            更新档里包含了对存档Bug的修复。
            问:是繁简通用的，还是说只能用在繁体版上？  答:DOS版就我所知只有繁体版，exe内写死了。(palxex)
            问:还更新了哪些？修复了哪些？  答:还有设定上的修改，根据外塞和夜烟的讨论。但没人总结过。(palxex)
            问:光盘修正版还需不需要打补丁了？ 答:不需要。(palxex)
            问:磁盘版和光盘版通用？  答:该补丁同时带有光盘版和磁盘版的修正。(palxex)
            需确认:对光盘修正版使用补丁会造成错误？详细后果？  答:会造成存档永久性错误。只影响存档的人物等级一带。(palxex)
            问:两个补丁各带一个存档修复工具CVSAVE.exe和crpg.exe，有何区别？  答:cvsave修正初版存档的经验值定义以配合修正版主程序，crpg修复初版所造存档的所有事件为未发生状态，不知何用。crpg使用多少次都是同样的恢复，所以不需要检测。(palxex)

        4)已知Bug
            1)存档经验值Bug。官方补丁修复。(磁盘，光盘初版确认有此Bug，光盘修正版无。(palxex))
            2)程序Bug两个，一个是灵壶咒的减少经验值效果写在了炼丹里面导致失效，另一个是0000脚本问题。(palxex)
            3)战外可使用金刚符之类道具。(palxex)
            4)装备道具并卸载特殊效果其后一战不消除。(palxex)
            5)无论是初版还是修正版，探测内存都会出问题，超过32M会检测为负值。游戏的安装配置程序install因为是Turbo Pascal7编译，有其特有的高频CPU(大约400MHz以上)上无法运行的问题。前者在仙痴破解修正版时修正，后者由ChanceClever修正。(palxex)    原因，因为DOS版用的是XMS规范较早的版本(1.0?)，该版规范中使用16位值表示XMS的大小(以KB为单位)，这样就限制了所能检测的XMS内存仅为64MB；再加上由于QBasic中不存在无符号整数，DOS版就直接使用16位有符号整数，结果导致了在XMS在32MB~64MB的时候，产生了负值。(SuperMouse)
            6)巫抗0的家伙还有10%机会逃脱打击，见http://post.baidu.com/f?kz=177777764 (palxex)
            7)傀儡状态下可以按F键使用法术。(palxex)

           参见:外塞历代修正补丁的记录及palxex的补完、仙盟桃花林讨论帖。
           问:还有哪些Bug？存在于哪个版本?

    2.大陆发行版
        1)大陆未发行过磁盘版
        2)光盘版
            NT810引进版，直接台版引进，与台版稍有区别，包装内盒为白，盘盒卡纸只有前后两张彩页。
            双语公司代理版，￥120，两种盘面，黄色调和粉色调。
            问:大陆发行的是繁体版？DOS版没有简体的？  答:从网上下载及各处讨论看，99%是没有官方简体版。
            问:如果有简体版，据win版的简体版中有许多繁体版中没有的文字错误，那DOS版是否也是这种情况。  答:同上


二、《仙剑奇侠传》Windows版
    1.台湾发行版(1996年6月)
        NT810版，内含部件包括游戏光盘，游戏说明书，主角彩色图卡4张，封面海报1张。
        认证方式为读光盘上的1.avi。(出自熊猫的minicd贴)
        含音轨，12首，曲目如下
            01-情怨
            02-蝶舞春园
            03-雨
            04-白河寒秋
            05-蝶恋
            06-云谷鹤峰
            07-桃花幻梦
            08-比武招亲
            09-梦中大侠
            10-拜月欺天
            11-君莫悲
            12-再续未了缘

    2.大陆发行版(1997年10月(palxex))
        双语公司代理版，￥138，在大陆发行的时候，被更名为98柔情篇，故台湾95版=大陆98版，柔情篇这一版本说法在台湾是不存在的。
        大量的低价普及版，包括晶合时代01年发行的晶合时代风版，盘面图案和台湾95版相同，并附使用手册、仙剑攻略和仙剑诗集，定价28元；而晶合在此之前还曾发行过红色包装版本，定价25元，内含部件为手册+光盘，数量极其稀少。
        另一种低价版本是百年树人软件发行的世纪回顾纪念版，该公司在世纪之交之即推出了大量经典游戏的低价回顾版本，但此版曾被大众软件指为未经大宇授权，擅自制作的伪正……回顾版分为3版，第1版附送4张人物彩色图卡，盘面为一贯的仙剑主题图片，印有世纪回顾纪念版字样。第2版无图卡。第3版盘面为粉色。
        还大量流传一种25元版本，其出处不详………

        问:除了转码所带来的文字错误，简体版比繁体版还多了哪些问题，收藏是否以繁体版为好？
        答:主要是文字造成的各种问题，其他未发现。事实上仙剑全系列，我是指仙剑DOS，Win95，SegaSaturn版，都以繁体为宜，无论质量，兼容，周边，都是繁体版较有保证，因为大宇自己在台澎金马作发行商。(palxex)

    3.官方补丁
        jorpark上只有win版的一个补丁。
        问:繁简通用的么？  答:只繁体需要，简体版出盘前即已修正。(SuperMouse)
        问:都补了些什么？

    4.与DOS版区别
        1)添加6个avi动画，分别是大宇商标、片头动画、梦中李逍遥、水魔兽消灭、灵儿与水魔兽同归于尽、结局的人物诗词。
        2)开头增加了逍遥梦见罗刹鬼婆的剧情。
        3)增加了物品说明。
        4)重新绘制少量迷宫地图，稍为漂亮了。
        5)Bug若干，如Midi音乐Bug，初始菜单黑屏，NPC跳河等等。

        参见:百度·仙剑98柔情篇吧、
        问:除了这些还有什么区别么？
        问:Win版的Midi音乐重新编排过了么，是否与DOS不同，是否不如DOS版出色？  答:比较音素是没发现什么，但应该作过处理，因为98版MIDI出问题了。(palxex)
        问:Win版的光盘音轨与DOS版光盘音轨有区别么？  答:除增加4首外，应该是没有。(palxex)

    5.已知Bug
        1)98原版震动大地脚本不起作用，SuperMouse完美补丁2.0里修正了。


三、《仙剑奇侠传》Sega Saturn版
    发行时间：1999年
    通过大宇和日本方面的频繁接触，仙剑走上了次世代的舞台，应该是次世代游戏机上第一款中文游戏。SS版包括游戏光盘一张，游戏手册一本，发行了中，日两种语言版本。此版剧情以WIN版为蓝本，但在插图，音乐，音效方面皆有变动，尤以游戏中的人物肖像变动最大。另外大宇在99年发行过SS版和95版通用的仙剑官方攻略。
    英文名：Legend Of Sword And Fairy
    日文名：Senken Kikyouden

    注：SS有模拟器，在SSF R11下测试，默认设置下黑屏，在Option--EZ Setting中设为Highest Compatibility时可玩。


四、《新仙剑奇侠传》
    1.台湾发行版
        2001年7月初版，分“挚爱月如版”和“深情灵儿版”，采用轩辕剑3引擎制作，包括4张游戏光盘，说明手册和手帕1条，两版内容完全相同，区别仅在于外包装封面及内赠手帕不同。收录了许如芸演唱的主题曲—有你的天堂，这是仙剑第一次出现真人演唱的歌曲。
        2002年1月仙剑奇侠传永恒回忆录版，是限量3000套的纪念版，包括新仙剑游戏光盘4张，游戏手册1本，DOS版音乐CD 1张，DOS版剧情攻略VCD 2张，原始人物彩色影印画27张，四人团圆版拼图1套，定价NT999。不过其中的剧情攻略VCD制作比较粗糙，不太符合典藏之说。

    2.大陆发行版(2001年7月15日(palxex))
        中青旅创先软件公司代理，先上市的是以挚爱月如版为封面的69元版本，后又发行了78元的灵儿版（封面不同于台湾的深情灵儿版），并推出过35元的真情回馈版，十分遗憾的是其中皆未收录主题曲。
        大陆版新仙剑的版本状况极为混乱，已知版本大致如下：
            1)￥69初版，外包装为套盒，上下两端未封口，背面印有“随盘附赠人物图卡 诚挚献礼真心传情”，说明书全彩，附送4张人物诗卡，仙剑主题贴纸。
            2)￥69二版，无人物诗卡，其余同首发。
            3)￥69三版，无人物诗卡，贴纸，包装盒上下封口，背面改为“岁月流逝经典永恒 诚挚献礼真心传情”，说明书封面内容全黑白。
            4)￥78初版，背面字样为“随盘附送传情手帕及人像钥匙扣 诚挚献礼真心传情”，说明书全彩，附送灵儿手帕，灵儿钥匙扣及4张人物诗卡（同69元版）。
            5)￥78二版，背面字样同首发，说明书封面彩色，内容黑白且有大量漏字现象，赠品中无人物诗卡。
            6)￥78三版，背面字样为“随盘附送传情手帕或人像钥匙扣 诚挚献礼真心传情”，赠品为手帕及钥匙扣随机一样，但据说90%为钥匙扣，说明书封面为69元版外包装图案，内容全黑白。
            7)￥35回馈版，说明书封面彩色，内容黑白，说明书封面同首发。
            8)￥69伪正版，说明书全彩并附送主题贴纸，整体相似度99%，区别仅在于回函卡上无抽奖内容，而且包装盒侧面下角没有No.28的字样。
            目前市面上销售的69和78版本皆为第3版，以时间来看应属于未授权的伪正。

    3.官方补丁
        jorpark上有繁简两个补丁。
        问:都更新啥了？  答:有程序、设定、迷宫、对话各方面，新仙剑初版bug极多。(palxex)

    4.与老仙剑区别
        1)开头梦境不一样 被改为战斗模式
        2)多了“逍遥神剑”一招
        3)画面分辨率提高
        4)人物对白有改动
        5)过场画面不一样，变的更加精细
        6)招式效果3D化色彩较重
        7)物品有3D化配图
        8)战斗音乐增加（好象？？）
        9)操作八方向，有别于老《仙剑》的四方向
        10)购买物品可调（老《仙剑》可望而不可及的）
        11)敌人有合击，会自己加血（这一点使新《仙剑》战斗难度增加了）
        12)战斗时有镜头缩放
        13)招式属性标明了
        14)换武器后人物手持母武器相映变化
        15)酒剑仙使用剑术后，地面剑痕保留
        16)姥姥变为蛇妖时变了两次
        17)有3结局，有两个是隐藏的
        18)头像与场景重绘(新头像褒贬不一，很有争议)
        19)增加新系统，紫金葫芦的炼药和“蛊物炼化”，玩家根据需要炼化出自己心怡的物品，如金蚕王、隐蛊等高价难求之物
        20)增加剧情，罗刹洞(梦境)、罗刹洞(回魂仙梦)，最后一战在殿前对拜月后增加了水下对拜月与水魔兽的合体怪兽，阿奴的登场时间也提前到了余杭镇码头(第一次从仙灵岛回后)，总之将98版中因故没有描写的部分加入了其中。
        问:还有么？

    5.已知Bug
        1)行走缓慢，不支持鼠标操作。
        问:还有么？


五、《仙剑客栈》(2001年8月)
    1.台湾发行版
        NT599，包括两张游戏光盘，游戏手册，并赠送精美餐具组一套（筷子+汤匙）。
   
    2.大陆发行版
        中青旅代理，￥58，双CD，说明书封面彩色，内容黑白，附送人物诗卡4张。
        伪正，双CD，说明书全部黑白，无诗卡。此作也有25元的真情回馈版，内容同首发，无赠品卡片。

    3.官方补丁
        jorpark上有繁体简体各一个补丁。

    问:简体繁体版可有区别？
    问:有一种单CD版卖，比较文件发现内容与双CD版相同，这是什么发行版？


六、《新仙剑奇侠传电视剧XP版》
    1.台湾发行版(2005年?月)
        炒冷饭之作，等同于《新仙剑奇侠传》+XP弹出补丁，外包装盒图案和游戏盘面改为了TV版主角头像，附说明书和一张产品注册卡。首发和再版在盒子背部大宇LOGO处的颜色有细微差别，定价NT399。

    2.大陆发行版(2007年4月)
        寰宇之星代理发行，售价49元，光盘四张，附送一套人物杯垫(3片)。

    问:还用不用打补丁了？(从主文件时间上看，比补丁要晚，但内容不同)

```

**Comment by miaocanada** [comment]
忘记说了，http://palxex.ysepan.com/ 的网友上传区有本人上传的一个7mb的试玩版，主程序为palt.exe，无法存档，不太清楚这算不算火星，是从PTT论坛上扒下来的。重复上传了两次因为第一次忘记加备注了，请删除第一次上传。
如果火星了请删除。
原帖地址：https://www.ptt.cc/bbs/Old-Games/M.1661868407.A.B23.html

**Comment by palxex** [comment]
非常感谢。经简单对比该版本我从未见过。

**Comment by miaocanada** [comment]
这是对于游戏历史学家来说的重磅好消息！
官方版《风起云涌》战斗音乐MIDI文件终于出土了！再也不是只有RIX文件了！
今天花了点时间用Dosbox+VirtualMIDISynth+Doomworld论坛玩家的Roland SC-55的soundfont文件，配置好之后，然后快速玩了玩那个7mb试玩版，用install.exe配置了MIDI，发现仙灵岛的首场战斗音乐是MIDI版的《风起云涌》！也就是说当年林坤信做了《风起云涌》的MIDI版，但不知道为何在上市的时候删掉并替换成了《势如破竹》。
我感觉里面可能还有其他不为人知的东西，现在只发现了这么多。

**Comment by palxex** [comment]
确实是好消息！此前玩家社区里一直传说那几首复制粘贴的midi是来不及做了，看来应该是另有隐情。
用[OopsWare兄的Pal95Tools](../tree/master/Pal95Tools)简单过了一遍，文件上全法术动画FIRE、全地图标志物MGO、全游戏对象名字WORD完全一致，此外M.MSG里有后面全部敌人的战斗对话（而剧情内容被如下删减了），证明该试玩版应该是最终版完工后直接删减下来的，具体是将全地图(MAP/GOP）/全人物头像RGM/全RNG动画等删减至只剩余杭和仙灵岛出场的。
DATA.MKF里比起正式版就多了个檀香，此外密码图不同，因为我没折腾过这块无法查证到底不同在哪里。
实际试玩过程只到乘船上苏州起行，新仙剑试玩版的流程应该是就此致敬（不过加了一小段到苏州城外见到林）。
比起明显更早的资讯展演示版（ysepan那边跟这个演示版放一起了），这个版本能用sdlpal直接玩，也直接解决了其无法存档的问题，但（疑因脚本裁剪）无法与正式版互换存档。

**Comment by miaocanada** [comment]
>需确认:对光盘修正版使用补丁会造成错误？详细后果？  答:会造成存档永久性错误。只影响存档的人物等级一带。(palxex)

抽了点时间看了看，个人认为应该是这样的：
补丁中的pal.exe，pals.exe 和 data.mkf 与光盘修正版中的同文件的binary是一致的，除非我用的再版CD-ROM镜像有问题。我用的CD-ROM镜像来源自仙剑英雄网那个混合CD音源版 http://www.palhero.net/?p=6282 ，安装后直接从安装目录将CD镜像拖走。所以对光盘修正版使用官方补丁的话可能是无效的，因为根本就是同样的三个文件，除非我用的CD-ROM镜像被改过了或者存在多个版本的再版CD-ROM。

初版CD-ROM的话，我手头上只有一个根据初版CD-ROM制作的盗版（从一个97年的盗版盘中拖走的，只有一个pal.exe被crack过然后没有pals.exe，运行这个被crack过的pal.exe的时候提示CD-ROM指向盘符j:\），其他内容应该没有删减，然后打了官方补丁后发现密码图选择色块和我那再版CD-ROM镜像版不一样。估计密码图的assets不光在data.mkf里，其他位置应该也有。

补丁后的初版
![palold](https://user-images.githubusercontent.com/7980264/192448270-41dcdae2-cffd-40b6-9a91-e4551402e2f3.JPG)

再版
![palnew](https://user-images.githubusercontent.com/7980264/192448339-3d1a13f7-755c-41b8-aa0c-115c7f4b6ea1.JPG)


关于那个试玩版中
>DATA.MKF里比起正式版就多了个檀香，此外密码图不同，因为我没折腾过这块无法查证到底不同在哪里。

试玩版的data.mkf 和那个初版盗版没打补丁时候的 data.mkf 的 binary 是一致的，也就是说试玩版应该从做完的初版直接拿的。另外尝试直接将补丁中的pals.exe放到那个试玩版里运行，logo之后只能看到顶部闪烁的一个花屏，于是又拖进去补丁中的data.mkf到试玩版里面得到密码图文字+选择框+花屏。
![paltmod](https://user-images.githubusercontent.com/7980264/192448681-9d8c31e2-501e-4273-be25-7411a56c2b76.JPG)



盲猜试玩版里密码图的assets不全。

**Reply by palxex** [reply]
> > 需确认:对光盘修正版使用补丁会造成错误？详细后果？  答:会造成存档永久性错误。只影响存档的人物等级一带。(palxex)
> 
> 抽了点时间看了看，个人认为应该是这样的： 补丁中的pal.exe，pals.exe 和 data.mkf 与光盘修正版中的同文件的binary是一致的，除非我用的再版CD-ROM镜像有问题。我用的CD-ROM镜像来源自仙剑英雄网那个混合CD音源版 http://www.palhero.net/?p=6282 ，安装后直接从安装目录将CD镜像拖走。所以对光盘修正版使用官方补丁的话可能是无效的，因为根本就是同样的三个文件，除非我用的CD-ROM镜像被改过了或者存在多个版本的再版CD-ROM。

当时回答的语境是：对初版存档使用更新档中的cvsave，将其修正到更新版exe可用的scheme。不作这个修复直接用更新版exe带老存档，所有人的等级、经验都会乱掉；修复一次以上，同样会乱掉。原因是初版生成的存档压根没有李逍遥经验的位置，更新版需要在其他人经验前把他的硬插进去，很显然，只能插一次。

> 
> 我手头上只有一个根据初版CD-ROM制作的盗版（从一个97年的盗版盘中拖走的，只有一个pal.exe被crack过然后没有pals.exe，运行这个被crack过的pal.exe的时候提示CD-ROM指向盘符j:\），其他内容应该没有删减

这个版本可以上传一下吗？我手头已经没有初版了

**Reply by miaocanada** [reply]
@palxex 已打包上传，原安装包在INSTALLER文件夹中。
其余文件（密码图+官方补丁+说明）都是我自己加的。

**Reply by miaocanada** [reply]
FYI，永硕盘现在有网关错误，貌似在维护。

**Reply by palxex** [reply]
非常感谢。碰到那边有问题时这边其实也可以上传，不占仓库容量。

**Reply by miaocanada** [reply]
对了还有个事情问一下，大宇资讯在iOS/iPadOS的App Store上架的“仙剑奇侠传1DOS怀旧版”在2.0之前是不是用的SDLPAL？我记的以前好像不是，从2.0之后才开始用的吧。

**Reply by palxex** [reply]
2.0之前确实不是sdlpal，是模拟器+修正版资源+iso（没留数据轨，只当CD音乐来源）的实现。模拟器本体印象中是iDOS/dospad，不过不是太确定，只能确定是个dosbox的变种。

**Reply by miaocanada** [reply]
一个脑洞：如果根据SDLPAL反向制作一个素材包/将官方的Assets资源全部替换然后将这个素材包自由软件化（使用GNU Public License等），有没有实现的可能？（当然副作用是素材包可以兼容PAL/PALS/PALT，例如做一个《歪瓜裂枣传》什么的，见笑了，类似的想法源自于Freedoom https://freedoom.github.io/ ）

**Reply by palxex** [reply]
乐见其成，不过我大概是没什么时间了。
这个项目本身有一些可能对这个目标有用的工具（比如PackageUtils和MapEditor），现在向原版资源包导入外部资源（包括各种图片、动画、脚本等）理论上并没有什么问题，但一直严重缺乏整合性。

**Reply by miaocanada** [reply]
好像这样做不太行，因为有法律问题，刚刚看了关于Total Conversion法律相关的东西，除非已经得到了大宇资讯官方的批准。按照美国的法律这样做属于“基于原有游戏的修改”法庭上是站在发行商这一边的。https://kblroche.com/thinking-before-modding-players-dont-own-what-they-make-2656426399.html

Freedoom的情况挺特殊的，一方面Doom引擎本身就被Id Software以GPL形式放出，另一方面可能是发行商的默许，但就算这样论坛上好多还在讨论Freedoom是否合法的问题。https://www.doomworld.com/forum/topic/102630-stance-of-the-original-id-software-team-on-freedoom/

**Comment by miaocanada** [comment]
哦各位注意一下，我上传的那个从盗版盘上扒下来的仙剑初版的installer有bug，直接运行的话可能在部分DOS环境以及DOSBox环境下直接提示“安装完成”但其实什么任务也没执行（当时这张盗版盘就很奇怪，里面的游戏有的时候能安装有的时候不能），最保险的办法是将.bat里面的一行命令直接执行（请把各个分卷都放在一个目录下）：

`DEARJ x -vv -jyc PAL.001`

2年前我记得直接在DOSBox下运行好像是没问题的，难道说跟时间戳有关系。不知道是.bat里面有问题还是DeArj本身的bug，感觉大概率是.bat写的有问题。
如果用WinRAR解压，需要将第一卷的拓展名改为.zip或者.arj，解压的时候会提示插入第二卷，把文件指向PAL.002（第二个分卷）即可。
